tweetset_server.py

```diff
@@ -20,6 +21,7 @@
 app = Flask(__name__)
 app.config['SECRET_KEY'] = os.environ.get('SECRET_KEY', 'secret')
 app.config['DATASETS_PATH'] = '/tweetsets_data/datasets'
+app.config['FULL_DATASETS_PATH'] = '/tweetsets_data/full_datasets'
```

Comment:
@dolsysmith @lwrubel Should this path be hard-coded, or should it take the path from the environment?

Reply:
The path in the `.ENV` is mapped to the Docker volume, and this line refers to the Docker volume path. I was keeping it parallel with the line above.

Reply:
Got it, thanks